### ISSUE TITLE
Squashing Commits Instructions unclear in README

### ISSUE THREAD

**Issue by annalorimer** [wics-uw/website#253]
#### Problem:
- README doesn't address:
  - When during the pull request process commits should be squashed
  - Doesn't explain what is actually means to squash a commit
- Possible mistakes:
  - "git push --force" should be "git push origin BranchName --force"

**Comment by evykassirer** [comment]
hm I think after the first time you push, it'll know what "push --force" means (at least, I only have to say git push)

but great points! we haven't actually specified when they should be squashed, but I like the idea of it happening at the end because then you can see the difference between commits as you go and what things you fixed

**Comment by annalorimer** [comment]
For me it didn't know where to push to for "git push --force", although it's entirely possible there was something else wrong. 

I agree about doing it at the end.

**Comment by evykassirer** [comment]
ah, yeah

every time I do `git push` the first time it's like "did you mean `git push .......` and then I copy paste that and forget about what it was :p we could explain this better

**Comment by fboxwala** [comment]
We had similar problems with the page in the git workshop, and Elana added helpful info to it. Could copy paste from there to README or link there from readme?

**Comment by annalorimer** [comment]
Just went through the instructions again for a recent pull request. The paragraph is really dense and hard to parse. 

Also, the simple git push --force does not work. It needs to be git push origin BranchName --force.

**Comment by arshiamufti** [comment]
I'll take this on. I also propose we pull all the git related stuff from the README into a second file--it takes ages to find stuff on it.
